packages/expo-video/ios/VideoPlayer.swift

```diff
@@ -3,4 +3,6 @@
 import AVFoundation
 import ExpoModulesCore
 
-internal final class VideoPlayer: SharedRef<AVPlayer> {}
+internal final class VideoPlayer: SharedRef<AVPlayer> {
+  let contentKeyManager = ContentKeyManager()
```

Comment:
Looks like we don't need to initialize it for non-DRM content 🤔 
```suggestion
  lazy var contentKeyManager = ContentKeyManager()
```